service/cloudconfig/master_extension.go

```diff
@@ -0,0 +1,100 @@
+package cloudconfig
+
+import (
+	providerv1alpha1 "github.com/giantswarm/apiextensions/pkg/apis/provider/v1alpha1"
+	"github.com/giantswarm/azure-operator/client"
+	"github.com/giantswarm/azure-operator/service/key"
+	"github.com/giantswarm/certs"
+	k8scloudconfig "github.com/giantswarm/k8scloudconfig/v_2_0_0"
+	"github.com/giantswarm/microerror"
+)
```

Comment:
`Files` and `Units` look the same. They call `render*File` and `render*Unit` and return slice of results.

All `render*` are unified. They call corresponding `render*` from `render.go`.